pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -215,6 +217,11 @@ type Controller struct {
 
 	// Network name for the registry as specified by the MeshNetworks configmap
 	networkForRegistry string
+
+	// initialSync becomes true only after the resources that exist at startup are processed
```

Comment:
what do you mean by `the resources that exist at startup`

Reply:
I think this is similar to HadSynced from the informer, but it also is checking we have processed them in our queue as well. So every known resource has been processed.